src/middlewares/Fetch.spec.ts

```diff
@@ -2,20 +2,27 @@ import {IRequest, IRequestHeaders} from "../Client";
 import {Fetch} from "./Fetch";
 
 describe("Fetch", () => {
+    describe("is defined and is available", () => {
+        it("should be defined", () => {
```

Comment:
I'd prefer to not wrap the "should be defined" it(). If we have this test, we should have it as the first test without any wrapping for the following reason:

it will pop up in the list first. In the case it fails, all other cases will logically fail as well. Having this test as first, will help the developer to not be confused about the reason of this error as this test basically proves the existence of a working export/import statements relationship. 

We could as a group agree to skip this test entirely, though i would prefer to have this two-line in by default as it will add confidence in the tech-stack used. 

PS: For everyone writing tests, it's probably a good habit to have a template already including this test in the given structure.